spec/views/users/user_index_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe 'users', type: :request do
+  let!(:user) do
+    User.create(
+      name: 'test user1',
+      photo: 'https://example.com/default-photo.jpg',
+      bio: 'test_bio',
+      posts_counter: 2
+    )
+  end
+  describe 'GET /index', type: :request do
+    before(:example) { get '/users' }
+
+    it 'displays a list of users' do
+      expect(response).to have_http_status(200)
+    end
+
```

Comment:
- [ ] It would be better if we implement the required test as seen in the screenshot below
![Screenshot from 2023-09-05 17-13-18](https://github.com/Peter-Dumbari/Blog-Application/assets/46388113/a3f50b55-541a-4471-843d-e3ad17c4d460)
However, i have looked through the test cases and i'm unable to find these